tests/ci/integration/run_trousers_integration.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash -exu
+#
+# Copyright Amazon.com Inc. or its affiliates.  All Rights Reserved.
+# SPDX-License-Identifier: Apache-2.0 OR ISC
+#
+
+source tests/ci/common_posix_setup.sh
+
+# Set up environment.
+
+# SYS_ROOT
+#  |
+#  - SRC_ROOT(aws-lc)
+#  |
+#  - SCRATCH_FOLDER
+#    |
+#    - trousers
+#    - trousers-install
+#    - AWS_LC_BUILD_FOLDER
+#    - AWS_LC_INSTALL_FOLDER
+
+# Assumes script is executed from the root of aws-lc directory
+SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+SCRATCH_FOLDER=${SYS_ROOT}/"TROUSERS_SCRATCH"
+TROUSERS_SRC_FOLDER="${SCRATCH_FOLDER}/trousers"
+TROUSERS_INSTALL_FOLDER="${SCRATCH_FOLDER}/trousers-install"
+AWS_LC_BUILD_FOLDER="${SCRATCH_FOLDER}/aws-lc-build"
+AWS_LC_INSTALL_FOLDER="${SCRATCH_FOLDER}/aws-lc-install"
+
+mkdir -p "${SCRATCH_FOLDER}"
+rm -rf "${SCRATCH_FOLDER:?}"/*
+
+pushd "${SCRATCH_FOLDER}"
+
+function trousers_build() {
+  #git apply "${SCRIPT_DIR}/trousers_patch/aws-lc-trousers.patch"
```

Comment:
Looks like we don't need a patch to build trousers against AWS-LC. Is this left in here in case that changes in the future?

Reply:
Yeah, I anticipated applying a patch, but it's not needed. I'll remove the line.